src/sidebar/components/test/stream-content-test.js

```diff
@@ -44,9 +40,7 @@ describe('StreamContentController', function () {
     };
 
     fakeApi = {
-      search: sinon.spy(function () {
-        return Promise.resolve({ rows: [], total: 0 });
-      }),
+      search: sinon.stub().resolves({ rows: [], replies: [], total: 0 }),
```

Comment:
This updates the fake API response to include the `replies` field, which was missing before.